book/src/how/r1cs_constraints.md

```diff
@@ -21,7 +21,7 @@ The inputs required for the constraint system for a single CPU step are:
 
 #### Pertaining to read-write memory
 * The (starting) RAM address read by the instruction: if the instruction is not a load/store, this is 0.
-* The bytes written to or read from memory.
+* The bytes are written to or read from memory.
```

Comment:
Either leave as is, or change to "The bytes that are written to or read from memory"